pkg/apis/redhatcop/v1alpha1/keepalivedgroup_types.go

```diff
@@ -26,6 +26,9 @@ type KeepalivedGroupSpec struct {
 
 	// +kubebuilder:validation:Optional
 	VerbatimConfig map[string]string `json:"verbatimConfig,omitempty"`
+
+	// +kubebuilder:validation:Optional
```

Comment:
please add the `kubebuilder:validation:UniqueItems` validators rule here.
also please see if these validators: `kubebuilder:validation:Minimum` and `kubebuilder:validation:Maximum` can be used for list.

Reply:
uniqueitems added, min, max doesn't work with list and there is no equivalent available.

Reply:
I'll have to remove the unique:
```
$ oc apply -f redhatcop.redhat.io_keepalivedgroups_crd.yaml
The CustomResourceDefinition "keepalivedgroups.redhatcop.redhat.io" is invalid: spec.validation.openAPIV3Schema.properties[spec].properties[blacklistRouterIDs].uniqueItems: Forbidden: uniqueItems cannot be set to true since the runtime complexity becomes quadratic
```

Reply:
ok, weird...thanks for testing this...